test/unit/sql/data-types.test.js

```diff
@@ -233,6 +233,41 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
       });
     });
 
+    describe('TIME', () => {
+      testsql('TIME', DataTypes.TIME, {
+        ibmi: 'TIME',
+        postgres: 'TIME',
+        mssql: 'TIME',
+        mariadb: 'TIME',
+        mysql: 'TIME',
+        db2: 'TIME',
+        sqlite: 'TIME',
+        snowflake: 'TIME',
+      });
+
+      testsql('TIME.ZONED', DataTypes.TIME({ zoned: true }), {
```

Comment:
```suggestion
      testsql('TIME({ zoned: true })', DataTypes.TIME({ zoned: true }), {
```

Reply:
You misunderstood, i make three testsql:

- DataType.TIME
- DataType.TIME({ zoned: true })
- DataType.TIME.ZONED

I will make new commits, if third test is unnecessary I can remove it

Reply:
I know, but two of these tests are named `TIME.ZONED`, this one should be named `TIME({ zoned: true })` as that's what is being tested (the first argument is the name of the test)

Reply:
Understand, I renamed it

Reply:
perfect :)